stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/components/MessageInput.kt

```diff
@@ -109,6 +109,7 @@ public fun MessageInput(
     InputField(
         modifier = modifier,
         value = value,
+        maxLines = 6,
```

Comment:
I feel like this should be customizable and a constant somewhere - like a parameter of the MessageInput, that is `DEFAULT_MESSAGE_INPUT_LINE_LIMIT`, that we define within our SDK